library_generation/generate_repo.py

```diff
@@ -30,11 +32,12 @@ def main(ctx):
 @main.command()
 @click.option(
     "--generation-config-yaml",
-    required=True,
+    required=False,
     type=str,
     help="""
     Path to generation_config.yaml that contains the metadata about
-    library generation
+    library generation. If not specified, it will be assumed to be found
+    at the root of --repository-path
```

Comment:
What's the name of default generation config?

Reply:
Is it ok to leave this as a convenience for testing the generation scripts without the PR description generation side of things? This may save a bit of setup (e.g. not creating a baseline and current gen config + avoiding the overhead image build part of the integration test)

Reply:
In `entry_point.py`, there's an option to only provide one configuration and skip pr description generation.